ycms/cms/templates/_raw.html

```diff
@@ -46,33 +46,50 @@
                  class="absolute top-32 right-8 flex flex-col max-w-screen-sm z-50">
                 {% for msg in messages %}
                     {% if msg.level_tag == 'info' %}
-                        <div class="message bg-blue-100 border-l-4 border-blue-500" role="alert">
+                        <div class="message bg-blue-100 border-l-4 border-blue-500 dark:bg-gray-700 dark:border-blue-400"
+                             role="alert">
                             <p class="text-blue-500">{{ msg.message }}</p>
```

Comment:
Hm... It is definitely more readable now, but I kinda miss the colored backgrounds :(

Making the text color important works to override the `p` styling:
```suggestion
                        <div class="message bg-blue-100 border-l-4 border-blue-500 dark:bg-gray-700 dark:border-blue-400"
                             role="alert">
                            <p class="!text-blue-500">{{ msg.message }}</p>
```
Or do you think the green backgrounds are too bright for dark mode?

Reply:
Both are fine for me, I'm not sensible to colors😅 I change the backgrounds to their respective darker ones in dark mode.

Reply:
Alright, looks great now!!